BCStudentSoftwareDevTeam/celts
# Send recommendation functionality

For issue #643 

Creates a CSV file of all approved courses for a specific term.

Opens up an outlook email with specified Subject and Body. 

**Waiting on CELTS to reply with what the body of the email needs to be**

To test:
- Navigate to course management.
- Click Download File under approved courses.
- Ensure file downloads and it has all approved courses for that term.
- Run tests to ensure downloadFile.py is operating correctly.
- Open email using "Send recommendation" button, ensure body and subject are prefilled.
